ami/ml/views.py

```
@@ -188,3 +192,74 @@ def register_pipelines(self, request: Request, pk=None) -> Response:
+    @extend_schema(
+        operation_id="projects_pipelines_create",
+        summary="Register pipelines for a project",
+        description=(
+            "Receive pipeline registrations for a project. This endpoint is called by the "
+            "V2 ML processing services to register available pipelines for a project."
+        ),
+        request=PipelineRegistrationSerializer,
+        responses={201: PipelineRegistrationResponse},
+        tags=["projects"],
+    )
+    def create(self, request, *args, **kwargs):
```

Ruff (0.14.14)

[warning] Unused method argument: `args`

(ARG002)